hapi/src/utils/lifebankcode.js

```diff
@@ -1,7 +1,7 @@
 const eosUtil = require('./eos')
 
-const CONTRACT_NAME = 'lifebankcode' // @todo: use ENV
-const COMMUNITY_ASSET = '0 LIFE' // @todo: use ENV
+const CONTRACT_NAME = process.env.LIFEBANKCODE_CONTRACTNAME
+const COMMUNITY_ASSET = `0 ${process.env.BLOOD}`
```

Comment:
You are using a env variable named BLOOD but its not defined in the .env.example probally you should use COMMUNITY_ASSET